arangod/Transaction/Context.h

```diff
@@ -134,10 +134,8 @@ class Context {
   TRI_vocbase_t& _vocbase;
   std::unique_ptr<velocypack::CustomTypeHandler> _customTypeHandler;
 
-  ::arangodb::containers::SmallVectorWithArena<arangodb::velocypack::Builder*,
-                                               32>
-      _builders;
-  ::arangodb::containers::SmallVectorWithArena<std::string*, 32> _strings;
+  absl::InlinedVector<arangodb::velocypack::Builder*, 4> _builders;
```

Comment:
As we are handling out `Builder` objects often, I suggest we increase the capacity to `8` here.
```suggestion
  absl::InlinedVector<arangodb::velocypack::Builder*, 8> _builders;
```